4-projects/modules/standard_projects/data.tf

```diff
@@ -14,20 +14,56 @@
  * limitations under the License.
  */
 
-data "google_projects" "prod_project" {
-  filter = "labels.application_name:org-shared-vpc-prod"
+data "google_projects" "base_prod_project" {
```

Comment:
Similar to the folder_environments, the standard projects module makes less sense in the new structure where we have sub folders per environment. I think we can remove it